### PR TITLE
Version 2.0.32_2 to fix #263

### LINKED ISSUE #263: fahrplan doesn't work in Germany
The app fahrplan doesn't work in Germany. No contact to the server of Deutsche Bahn. The app not suggests any cities.  It's not possible to enter cities or Lacations in the app.

### PR DESCRIPTION
**Pull request by dayrover**
As discussed this is Chris Clime Curl fix plus the 'openstore.fahrplan2' app ID and version 2.0.32_2 which is now available on the OpenStore.

### CODE REVIEW

File: src/gui/ubuntu/AboutPage.qml [hunk 1]
```diff
@@ -135,7 +135,14 @@ Page {
...
                 }
-
+                  
+                Label {
+                   width: parent.width
+                   textSize: Label.Small
+                   horizontalAlignment: Text.AlignHCenter
+                   text: "(temporary fix for bahn.de %1)".arg("<a href=\"https://github.com/balcy/fahrplan\">here</a>")
+                   onLinkActivated: Qt.openUrlExternally(link)
+                }
```
Comment (smurfy) [inline src/gui/ubuntu/AboutPage.qml]: please remove

File: src/gui/about.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ var support = qsTr("<p>If fahrplan is not translated in your language, help us <
         "<p>If you like fahrplan, I would appreciate a donation. <a href=\"https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=DW3LGJ6VEKFVS\">Donate using PayPal</a></p>" +
         "<p>If you encounter bugs or have feature requests visit the <a href=\"https://github.com/smurfy/fahrplan/issues\">Fahrplan issue tracker on GitHub</a>.</p>");
 
-var maintainers = ["Philipp Andreas (smurfy)", "Michael Zanetti (mzanetti)", "Oleksii Serdiuk (leppa)"];
+var maintainers = ["Philipp Andreas (smurfy)", "Chris Clime (balcy)", "Oleksii Serdiuk (leppa)"];
```
Comment (smurfy) [inline src/gui/about.js]: is this the correct name here?

File: src/gui/about.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ var support = qsTr("<p>If fahrplan is not translated in your language, help us <
         "<p>If you like fahrplan, I would appreciate a donation. <a href=\"https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=DW3LGJ6VEKFVS\">Donate using PayPal</a></p>" +
         "<p>If you encounter bugs or have feature requests visit the <a href=\"https://github.com/smurfy/fahrplan/issues\">Fahrplan issue tracker on GitHub</a>.</p>");
 
-var maintainers = ["Philipp Andreas (smurfy)", "Michael Zanetti (mzanetti)", "Oleksii Serdiuk (leppa)"];
+var maintainers = ["Philipp Andreas (smurfy)", "Chris Clime (balcy)", "Oleksii Serdiuk (leppa)"];
 var codeContributors = ["gri", "lorenzph", "CaCO3", "hcm", "thp", "qwerty12", "qbast", "Thomas Fischer", "Oleksii Serdiuk (leppa)", "Nekhelesh Ramananthan (nik90)", "NightShift79", "pirast", "Michael Zanetti (mzanetti)", "shentok", "evansebire", "unsanded", "Morpog", "benni0815", "Erik Lundin"].sort();
```
Comment (smurfy) [inline src/gui/about.js]: Please add yourself and balcy here too

File: src/parser/parser_abstract.cpp [hunk 4]
```diff
@@ -101,6 +112,16 @@ void ParserAbstract::cancelRequest()
 
 void ParserAbstract::sendHttpRequest(QUrl url, QByteArray data, const QList<QPair<QByteArray,QByteArray> > &additionalHeaders)
 {
+    // workaround for Ubuntu / HTTPS with bahn.de
+    // here we use cUrl instead
+    #ifdef BUILD_FOR_UBUNTU
+    if (url.host().endsWith("bahn.de"))
```
Comment (smurfy) [inline src/parser/parser_abstract.cpp]: don't like this much, but its fine with me for now. i think about something better and maybe add this myself later.

File: src/gui/ubuntu/AboutPage.qml (hunk 1 above)
Comment (dayrover) [reply src/gui/ubuntu/AboutPage.qml]: Whole Label removed

File: src/gui/about.js (hunk 3 above)
Comment (dayrover) [reply src/gui/about.js]: OK

File: src/gui/about.js (hunk 2 above)
Comment (dayrover) [reply src/gui/about.js]: This came from Chris's commit, so I guess it ist correct.

File: src/parser/parser_abstract.cpp (hunk 4 above)
Comment (dayrover) [reply src/parser/parser_abstract.cpp]: I would simple add a boolean parameter to an overload parse_abstract constructor which selects the use of Curl. Should be OK to use Curl for alle URL used by the mobilebahnde_parser.

Comment (smurfy) [reply src/parser/parser_abstract.cpp]: sounds good to me

**Pull request merged**